src/importexport/guitarpro/internal/gtp/gpbeat.h

```diff
@@ -37,7 +37,7 @@ class GPBeat
         None, FadeIn, FadeOut, VolumeSwell
     };
     enum class Hairpin {
-        None, Crescendo, Decrescendo
+        None, Crescendo, Diminuendo
```

Comment:
Let's not rename this one, as it seems to be intended to reflect the GP terminology